webview/src/plots/components/App.test.tsx

```diff
@@ -700,42 +700,6 @@ describe('App', () => {
     expect(screen.queryByText('Images')).not.toBeInTheDocument()
   })
 
-  it('should hide plots when their section is collapsed (setting to null can break some Vega plots)', async () => {
```

Comment:
I think this might have been there because we used to animate the height of the section. With the changes it re-renders everything when we open the section. If anything, it should solve any previous rendering problems.